### PR TITLE
ODIS monitor updates

### PR DESCRIPTION
**Pull request by eelanagaraj**
### Description
- Adds three new monitor functions (one per new API: LegacyPNP, PNP, Domains)
- PNP & LegacyPNP are basically the same as the old version of the monitor (i.e. they generate new PKs + send a request that is then expected to receive out of quota), but with updated SDKs querying the new endpoints
- Domains queries the same domain which is set up with a really high rate limit (realistically shouldn't expire if this is run every 5 min). This is so that it actually produces a signature that can then be verified client-side (i.e. checking that signers are configured properly) (see [this issue](https://github.com/celo-org/celo-monorepo/issues/9817))

### Other changes
- fixed config + env var stuff (old version of the monitor didn't actually set the network properly)
- updated load test script accordingly (for PNP/LegacyPNP)
- updated deployment commands (so as to not accidentally delete the old ODIS 1 monitoring function)
- updated some outdated stackdriver alerts (?) to not measure latency in the new monitor functions

### Tested
- deployed to staging and alfajores (only us-central1 instances), set up dashboard panels for this
- confirmed that load testing script still works

### Doesn't include
- have not yet set up alerts for ODIS 2

### CODE REVIEW

File: packages/phone-number-privacy/monitor/src/index.ts [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+  // .region('us-central1', 'europe-west3')
+  .pubsub.schedule('every 5 minutes')
+  .onRun(async () => testPNPQuery(blockchainProvider, contextName, CombinerEndpointPNP.PNP_SIGN))
+
+export const odisMonitorScheduleFunctionDomains = functions
+  .region('us-central1')
+  // .region('us-central1', 'europe-west3')
   .pubsub.schedule('every 5 minutes')
-  .onRun(testQuery)
+  .onRun(() => testDomainsQuery(contextName))
```
Comment (alecps) [inline packages/phone-number-privacy/monitor/src/index.ts]: should be async?

File: packages/phone-number-privacy/monitor/src/scripts/run-load-test.ts [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
+    blockchainProvider = 'https://forno.celo.org'
     break
   default:
     printHelpAndExit()
     break
 }
-process.env.NETWORK = args[0]
 
-concurrentLoadTest(Number(args[1])) // tslint:disable-line:no-floating-promises
+concurrentLoadTest(Number(args[1]), blockchainProvider!, args[0] as OdisContextName) // tslint:disable-line:no-floating-promises
```
Comment (alecps) [inline packages/phone-number-privacy/monitor/src/scripts/run-load-test.ts:31]: Ahh have you been running this against mainnet? Spruce noticed an increase in traffic

**Review by alecps — APPROVED** [review]
looks great!!

File: packages/phone-number-privacy/monitor/src/query.ts [hunk 3]
```diff
@@ -36,12 +49,33 @@ export const queryOdisForSalt = async () => {
...
+        // Running every 5 min, this should not run out for the next 9 million years
+        batchSize: defined(1000000000000),
+        repetitions: defined(1000000000000),
+      },
+    ],
+    environment: serviceContext,
+  }
+  const authorizer = odisQueryAuthorizer(Buffer.from('ODIS domains monitor authorizer test seed'))
+  const domain = buildOdisDomain(monitorDomainConfig, authorizer.address)
+  return odisHardenKey(Buffer.from('password'), domain, serviceContext, authorizer.wallet)
```
Comment (alecps) [inline packages/phone-number-privacy/monitor/src/query.ts:81]: might be worth having a comment explaining that this throws if the signature verification fails

File: packages/phone-number-privacy/monitor/src/scripts/run-load-test.ts (hunk 2 above)
Comment (eelanagaraj) [reply packages/phone-number-privacy/monitor/src/scripts/run-load-test.ts:31]: Nope just staging :-/ this version runs against the new combiner endpoint anyways so don't think it should have affected them!

File: packages/phone-number-privacy/monitor/src/index.ts (hunk 1 above)
Comment (eelanagaraj) [reply packages/phone-number-privacy/monitor/src/index.ts]: indeed!

**Pull request merged**